chandra_aca/star_probs.py

```diff
@@ -276,7 +439,8 @@ def spline_model_acq_prob(mag=10.0, t_ccd=-12.0, color=0.6, halfwidth=120, probi
         t_ccd, mag, color, halfwidth)
 
     if np.any(t_ccds < -16.0):
-        warnings.warn('\nSpline model is not calibrated below -16 C, so take results with skepticism!\n'
+        warnings.warn('\nSpline model is not calibrated below -16 C, '
+                      'so take results with skepticism!\n'
```

Comment:
If there are edits in here, maybe add that warn on the spline model for temps greater than -10 .

Reply:
Done.

Reply:
Well, it *was* done but then caused 3 warnings to show up in tests.  Since this ship has already sailed I just want to keep the tests clean and move on.  So I backed it out.